i18n/uk/white-paper.md

```diff
@@ -0,0 +1,583 @@
+Враховуючи, скільки пакетів із відкритим вихідним кодом доступно сьогодні та їхнє очікуване зростання, $\chi$ завжди буде дуже малим значенням від $0$ до $1$.
+
+Нехай $\psi$ буде коефіцієнтом ставки.
+Він представляє загальну кількість токенів tea, поставлених будь-яким учасником мережі для захисту мережі.
+
+Нехай $\chi_{ideal}$ буде коефіцієнтом занурення, якого ми хочемо досягти для кожного пакета для справедливого розподілу винагород між усіма пакетами та їх залежностями.
+Значення $\chi_{ideal}$ потрібно оновлювати, коли нові пакети додаються до децентралізованого реєстру та створюються залежності.
+Щоб визначити найкраще значення для $\chi_{ideal}$, ми будемо використовувати криву популярності, оновлену на початку кожного циклу винагороди.
+
+Нехай $\tau = \tau(\chi)$ буде річною відсотковою ставкою, розподіленою між усіма членами чайної спільноти, які займаються токенами чаю для підтримки розробників з відкритим кодом.
+Іншими словами, $\tau(\chi)$ відповідає винагороді за замочування, яку отримує протягом року учасник спільноти, який замочує токени чаю за весь рік.
+
+Нехай $\gamma = \gamma(\psi)$ буде річною відсотковою ставкою, розподіленою між усіма операторами вузлів і учасниками мережі, які роблять ставку на чайні токени для захисту мережі.
+Іншими словами, $\gamma(\psi)$ відповідає винагороді за ставку, отриману протягом року учасником спільноти, який ставить токени чаю за весь рік.
+
+Нехай $\delta$ буде річною інфляцією, спрямованою на скарбницю мережі.
+$\delta$ може змінюватися, оскільки зовнішні фактори впливають на пропозицію токенів.
+
+Ми розглядаємо річну ставку винагороди за просування як функцію $\chi$ і річну ставку винагороди за ставку як функцію $\psi$.
+
+* $\tau(\chi)$ відповідає стимулу для людей накачувати пакет.
+Зі збільшенням $\chi$ потрібно менше винагород $\tau(\chi)$.
+* $\gamma(\psi)$ відповідає стимулу для людей робити ставки в мережі.
+Зі збільшенням $\psi$ для захисту мережі потрібно менше винагород $\gamma(\psi)$.
+
+Річна інфляція $I$ буде еквівалентна $(\tau + \gamma + \delta)$ і розраховуватиметься таким чином:
+
+$$
+I = \frac{\textrm{пропозиція токенів на кінець року} - \textrm{пропозиція токенів на початок року}}{\textrm{пропозиція токенів на початок року}} = (\tau + \гамма + \дельта)
+$$
+
+Слід зважити внесок в інфляцію $\tau_{\textsc{all}}$ (стимул, розподілений між усіма учасниками пакетів) і $\gamma_{\textsc{all}}$ (стимул, розподілений між усіма учасниками безпеки мережі) щоб переконатися, що система стимулює оптимальне співвідношення замочування/розбивки.
+
+Оскільки ми зосереджуємось на заохоченнях, розподілених між усіма пакетами, ми визначаємо це
+$\tau_{\textsc{все}}$
+є функцією коефіцієнта замочування $\chi$ і тому
+$\tau_{\textsc{усі}}(\chi) = \chi \cdot \tau(\chi)$.
+З нашого попереднього аналізу ми це бачимо
+$\tau_{\textsc{все}}(\chi_{ideal}) = \chi_{ideal} \cdot \tau_{ideal}$.
```

Comment:
For consistency sake, please keep all formula subscripts in English.

`\textsc{all}` instead of `\textsc{все}`

The same goes for `ideal`. Please double check all formula in this document.